pages/examples/features/authentication/index.tsx

```diff
@@ -9,30 +9,36 @@ import Cookies from 'js-cookie';
 import GlobalModalManager from '@system/modals/GlobalModalManager';
 import Input from '@system/Input';
 import KeyHeader from '@system/KeyHeader';
+import ModalError from '@demos/modals/ModalError';
 import MonospacePreview from '@system/MonospacePreview';
 import Page from '@components/Page';
 import ThinAppLayout from '@system/layouts/ThinAppLayout';
 import ThinAppLayoutHeader from '@system/layouts/ThinAppLayoutHeader';
 
 import { P } from '@system/typography';
 import { FormHeading, FormSubHeading, FormParagraph, InputLabel } from '@system/typography/forms';
-import { useModal } from '@system/providers/ModalContextProvider';
+import { useModal } from '@root/system/modals/ModalContext';
 
 function ExampleAuthentication(props) {
-  const { showModal } = useModal();
+  const modalError = useModal(ModalError);
```

Comment:
seems like the right approach is just `const modal = useModal(ModalError)`, since modal is only used once

Reply:
i thought about this at first, but i also thought that if i wanted to use another modal alongside it in the future, i would have to refactor the existing name to differentiate it. which is not a big deal, but this just made it feel like neither option has any real advantage. 

should i change it?

Reply:
No thats fine, but I think you're more likely to go to `modals` then each individual `modal`, we can cross that bridge later I suppose.